api/audit-trail-gw/src/database/channel-info.ts

```diff
@@ -37,7 +39,9 @@ export const searchChannelInfo = async (channelInfoSearch: ChannelInfoSearch): P
 		created: created && { $gte: created },
 		latestMessage: latestMessage && { $gte: latestMessage },
 		'topics.source': regex(topicSource),
-		'topics.type': regex(topicType)
+		'topics.type': regex(topicType),
+		hidden: hidden === true || hidden === false ? hidden : undefined,
```

Comment:
this should be done in getUserSearch = (req: Request): UserSearch => {} of the identity route. you are also not considering it here